scripts/deploy.sh

```diff
@@ -10,7 +10,7 @@ export DEPLOY_OA=${DEPLOY_OA:-"yes"}
 export DEPLOY_ELK=${DEPLOY_ELK:-"yes"}
 export DEPLOY_MAAS=${DEPLOY_MAAS:-"no"}
 export DEPLOY_TEMPEST=${DEPLOY_TEMPEST:-"no"}
-export DEPLOY_CEILOMETER=${DEPLOY_CEILOMETER:-"no"}
+export DEPLOY_CEILOMETER="no"
```

Comment:
I think I get why this was done - so that any callers of this script like a gate check, don't accidentally try to test aodh/ceilometer when the configs are deleted.

This solution works for me, but will mean another edit if/when ceilometer is supported.